example-dvc-experiments/generate.bash

```diff
@@ -4,7 +4,7 @@ set -veux
 
 HERE="$( cd "$(dirname "$0")" ; pwd -P )"
 export HERE
-PROJECT_NAME="example-dvc-experiments"
+PROJECT_NAME="example-dvc-staging"
```

Comment:
should we rename it back?

Reply:
which one should be permanent? I usually create & use (private) `example-dvc-staging` more frequently than (public) `example-dvc-experiments` and pushing the example repository with the created script is very easy, and may be used mistakenly to push versions with bugs, etc. 

I don't have strong opinions here, the user must edit this line before generating the script. 

Can we rename the repository itself after building and pushing? We can rename `example-dvc-experiments` to `example-dvc-experiments-22-01-26` and archive, then rename `example-dvc-staging` to `example-dvc-experiments`, and make public. Is it too much work?